components/bl-list-component/src/index.js

```diff
@@ -0,0 +1,34 @@
+import { useRef } from "react";
+
+import { List } from "./list";
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function ListComponent({ component, eventHandlers }) {
+  const { classList, width, backgroundColor, type, listItems, color, fontSize, padding } = component;
+
+  const onClickHandler = (e, key) => {
+    e.stopPropagation();
+    eventHandlers.onClickListItem({ item: key });
+  };
+
+  const options = {
+    type: type && type === "ol" ? "ol" : "ul",
+    style: {
+      color,
+      fontSize,
+      padding,
+    },
+    onClickHandler: onClickHandler,
+  };
+
+  const listRef = useRef();
+
+  component.el = listRef.current;
```

Comment:
https://github.com/Backendless/ui-builder-library/pull/195#discussion_r1084203591

>I'm not sure this is the correct way
let's imaging this was rendered only once and the animationOnScrollRef.current before first render will be null and as result it won't work
I assume we need to wrap it with useEffect